cmd/tkn/main.go

```diff
@@ -15,18 +15,66 @@
 package main
 
 import (
+	"fmt"
 	"os"
+	"path/filepath"
+	"syscall"
 
+	"github.com/pkg/errors"
 	"github.com/tektoncd/cli/pkg/cli"
 	"github.com/tektoncd/cli/pkg/cmd"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 )
 
+const (
+	pluginDirEnv = "TKN_PLUGINS_DIR"
+	pluginDir    = ".config/tkn/plugins"
```

Comment:
```suggestion
	pluginDir    = "~/.config/tkn/plugins"
```

Reply:
Oh right, I wanted to use `homedir` to expand this :sweat_smile:

Reply:
Updated, using `homedir.Expand` now :wink: